data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -127,16 +126,22 @@ then
 		--manifest-kube-client-agent-image "${MACHINE_CONFIG_KUBE_CLIENT_AGENT_IMAGE}" \
 		--asset-input-dir /assets/tls \
 		--asset-output-dir /assets/etcd-bootstrap \
-		--config-output-file /assets/etcd-bootstrap/config
+		--config-output-file /assets/etcd-bootstrap/config \
+		--cluster-config-file=/assets/manifests/cluster-network-02-config.yml
 
 	# TODO: host-etcd endpoint rendered by cluster-etcd-operator
 	BOOTSTRAP_IP=$(hostname -I | awk '{ print $1 }')
 	ETCD_ENDPOINTS=https://"${BOOTSTRAP_IP}":2379
-	sed -i "/__BOOTSTRAP_IP__/${BOOTSTRAP_IP}/"  /opt/openshift/manifests/etcd-host-service-endpoints.yaml
+	sed -i s/__BOOTSTRAP_IP__/"${BOOTSTRAP_IP}"/ /opt/openshift/manifests/etcd-host-service-endpoints.yaml
```

Comment:
This looks wrong syntax. The double quote is missing before s/, and there is a redundant / after the sed expression.

Reply:
Well, never mind. I see what you did.

Reply:
$ man sed
gives this usage example `s/regexp/replacement/` I think the trailing '/' closing the replace is OK usage.

Reply:
yeah looks nicer with full quotes and still resolves shellcheck.  `Double quote to prevent globbing and word splitting. [SC2086]` Will update